bottlecap/src/config/mod.rs

```diff
@@ -109,6 +111,16 @@ pub fn get_config(config_directory: &Path) -> Result<Config, ConfigError> {
         return Err(ConfigError::UnsupportedField("appsec_enabled".to_string()));
     }
 
+    match config.extension_version.as_deref() {
+        Some("next") => {}
+        Some(&_) | None => {
```

Comment:
why &_ rather than _?

Reply:
I originally had it as a separate branch and had some logic, but removed it. Will replace!